src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -15,7 +33,91 @@ void EnBal_Destroy(Actor* thisx, PlayState* play);
 void EnBal_Update(Actor* thisx, PlayState* play);
 void EnBal_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+typedef enum EnBalEyeTexture {
+    /* 0x0 */ EN_BAL_EYETEX_OPEN,
+    /* 0x1 */ EN_BAL_EYETEX_CLOSED
+} EnBalEyeTexture;
+
+typedef enum EnBalAnimation {
+    /* 0x0 */ EN_BAL_ANIM_FLOAT_DRAW,
+    /* 0x1 */ EN_BAL_ANIM_FALL_LOOP,
+    /* 0x2 */ EN_BAL_ANIM_FALL_ONCE,
+    /* 0x3 */ EN_BAL_ANIM_LAND,
+    /* 0x4 */ EN_BAL_ANIM_TALK_TWIST,
+    /* 0x5 */ EN_BAL_ANIM_TALK_IDLE,
+    /* 0x6 */ EN_BAL_ANIM_MAGIC_FULL,
+    /* 0x7 */ EN_BAL_ANIM_HAPPY_DANCE_LOOP,
+    /* 0x8 */ EN_BAL_ANIM_HAPPY_DANCE_ONCE,
+    /* 0x9 */ EN_BAL_ANIM_MAGIC_SHORT,
+    /* 0xA */ EN_BAL_ANIM_WAIT,
+    /* 0xB */ EN_BAL_ANIM_SPIN,
+    /* 0xC */ EN_BAL_ANIM_HIDE_FACE,
+    /* 0xD */ EN_BAL_ANIM_CONFETTI
+} EnBalAnimation;
```

Comment:
```suggestion
    /* 0xD */ EN_BAL_ANIM_CONFETTI,
    /* 0xE */ EN_BAL_ANIM_MAX
} EnBalAnimation;
```
So then you can do:
`sAnimationInfo[EN_BAL_ANIM_MAX]`

(Also, would be nice to change `EN_BAL_ANIM_` -> `TINGLE_ANIM_`, similar for all these enums as well